Makefile

```diff
@@ -3,7 +3,8 @@ BIN_NAME=terraform-provider-instaclustr
 
 
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
-VERSION=1.14.10
+
+VERSION=1.14.11
```

Comment:
**[Required]** This is a feature addition, so I would assume it would be a good time for a minor version bump to `1.15.0`.